packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -78,7 +59,14 @@ const GitLabScopingSearchResults = (props: Props) => {
   >(
     graphql`
       fragment GitLabScopingSearchResults_query on Query
-        @argumentDefinitions(cursor: {type: "String"}, count: {type: "Int", defaultValue: 50})
+        @argumentDefinitions(
+          projectsFirst: {type: "Int", defaultValue: 20}
```

Comment:
@mattkrick in our 1-1, I mentioned that Igor and I ran into an issue where GitLab would return a server error depending on the `projectsFirst` value.

For me, if the `projectsFirst` number is 15 or lower and the sort order is `latest_activity_asc`, I get the server error. In the GQL explorer, the message is `Internal server error`. In our codebase, I can repeat the same result, but the error is `The user aborted a request`. If `projectsFirst` is greater than 15, it always works.

However, if I sort by `latest_activity_asc`, it doesn't matter what the `projectsFirst` value is, it always works. 

It seems unlikely that a borked Parabol project is causing the issue when `projectsFirst` is 15, because it's still included in the successful results when it's 16.

It seems like there's a bug in GitLab's API. Googling hasn't helped yet, but I'll keep trying tomorrow. 

Here's the query FYI. If I query anything other than `__typename`, I get the error:

```
query {
    projects(membership: true, first:15, sort: "latest_activity_asc") {
    nodes {
      createdAt
    }
  }
}  
```